addons/mission/functions/fnc_ping.sqf

```diff
@@ -34,15 +40,18 @@ _marker setMarkerBrushLocal "Border";
 _marker setMarkerSizeLocal [0, 0];
 _marker setMarkerColorLocal _colour;
 
+_location setVariable [QGVAR(pingInProgress), true, true];
```

Comment:
Why public if this function can only run on server anyways?

Reply:
Checking via ACE actions